lib/systems/include/Controllers/CASEControllerWrapper.h

```diff
@@ -5,14 +5,18 @@
 #include "CASESystem.h"
 
 template <typename message_queue>
+/// @brief makes CASE system a part of Controller hierarchy for use in TC Mux
 class TorqueControllerCASEWrapper : public virtual Controller
 {
 public:
     TorqueControllerCASEWrapper() = delete;
-    
+    /// @param case_instance provide current instance to ensure there are not duplicates of this system as it needs to stay updated for MCU to get data.
```

Comment:
its not to ensure that there are not duplicates of the system: while it does just give it a pointer to not copy the instance of the CASE system, it gets given this pointer to interact / control CASE. this pattern we refer to as "pointers as ownership"